### PR TITLE
javascript-javascript1-week1

### PR DESCRIPTION
**Pull request by Impressiv3**
new file:   index.html
new file:   style.css
new file:   index.js

### CODE REVIEW

**Comment by Infamou5** [comment]
HYF HOMEWORK - TASK 2 - Goodboy-Oldboy (A dog age calculator) - ADDED
HYF HOMEWORK - TASK 3 - Housey pricey (A house price estimator) - ADDED

**Comment by Infamou5** [comment]
HYF HOMEWORK - TASK 4 - Ez Namey (Startup name generator)  - ADDED

**Comment by Infamou5** [comment]
Cleaned up code using : JavaScript Formatter. Ready for review.

File: javascript/javascript1/week1/index.js [hunk 1]
```diff
@@ -3,29 +3,26 @@
 const yearOfBirth = 1983;
 const yearFuture = 2023;
 const age = yearFuture - yearOfBirth;
-const result = (`You will be ${age} years old in ${yearFuture}.`);
+const result = `You will be ${age} years old in ${yearFuture}.`;
 /* or const result = ("You will be " + age + " years old in" + yearFuture + ".") */
 
 console.log(result);
```
Comment (TITHI1244) [inline javascript/javascript1/week1/index.js:10]: Super nice and clean code. You could have added the variable data types in a comment as that was asked in the task :)

File: javascript/javascript1/week1/index.js [hunk 2]
```diff
@@ -3,29 +3,26 @@
...
 
 /* =======================================================================================*/
 
-
 /* HYF HOMEWORK - TASK 2 - Goodboy-Oldboy (A dog age calculator) */
 
 const dogYearOfBirth = 2000;
 const dogYearFuture = 2010;
 const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
 const shouldShowResultInDogYears = false;
```
Comment (TITHI1244) [inline javascript/javascript1/week1/index.js:19]: Same here. You could mention the variable data type as asked :)

**Review by TITHI1244 — APPROVED** [review]
Very well done. The code was clean with nice formatting. The workflow in git is very impressive. This is the supposed way of using git. Asked for adding 2 lines of comments as mentioned in the task. Keep coding :)

File: javascript/javascript1/week1/index.js (hunk 1 above)
Comment (Infamou5) [reply javascript/javascript1/week1/index.js:10]: I haven't payed attention to those questions, to be frank. 
1. Task - Age-ify (A future age calculator)
    yearOfBirth and yearFuture are both number types.
    
2. Task - Goodboy-Oldboy (A dog age calculator) 
   shouldShowResultInDogYears is a boolean type. 
   
There we go ! 

 Thank you for your time to review my code.

File: javascript/javascript1/week1/index.js (hunk 2 above)
Comment (Infamou5) [reply javascript/javascript1/week1/index.js:19]: Look @ the previous comment ;)  All set 'n done !

**Pull request merged**